test/integration/remi-php55/default_spec.rb

```diff
@@ -1,3 +1,25 @@
-describe command('php --version') do
+describe command('php55 --version') do
```

Comment:
Let's convert these individual tests into a proper InSpec profile that uses inputs instead. That way we have one code base that will work for all versions of php.

Reply:
do you have an example of this on hand? I havent made a profile before

Reply:
Take a look at the percona cookbook.